server.coffee

```diff
@@ -111,15 +119,29 @@ loadApp = (logger, loadedConfig) ->
     app.get "#{ APP_ROOT }/v1/health-check", (req, res) ->
       res.send('OK\n')
 
-    port = process.env.PORT ? loadedConfig.get('server.port').get() ? 5000
-    app.listen(port)
-
-    logger.log('Server listening on port %d in %s mode', port, app.settings.env)
+    if loadedConfig.get('server.https.options').get() instanceof Object
+      https = require 'https'
+      fs = require 'fs'
+      httpsOptions = _.mapObject loadedConfig.get('server.https.options').get(), (v, k) ->
+        if _.isString(v)
```

Comment:
So this is saying any option can also be a file? That seems very dangerous.  What if my string value also happens to be a file? I'm not a big fan of errors which appear and disappear based on what files show up in the local directory.